client/packages/common/src/ui/components/navigation/AppNavLink/AppNavLink.tsx

```diff
@@ -193,11 +192,10 @@ export const AppNavLink: FC<AppNavLinkProps> = props => {
                     selected || isSelectedParentItem ? 'bold' : 'normal',
                   color: isSelectedParentItem ? 'primary.main' : undefined,
                 },
-                flexGrow: 0,
               }}
             />
           </Badge>
-
+          <span style={{ flexGrow: 1 }} />
```

Comment:
Now we can clearly see there's a "spacer" here, as opposed to it being hidden inside the Badge component, which is invisible most of the time.